deploy/gcp/manifests/db.yaml.example

```diff
@@ -0,0 +1,63 @@
+apiVersion: pingcap.com/v1alpha1
```

Comment:
We have to convert the configurations in https://github.com/pingcap/tidb-operator/blob/master/deploy/modules/gcp/tidb-cluster/values/default.yaml to this file.

Reply:
I'll fix it.

Reply:
fixed, by the way, I didn't set storage class name for pd & monitor explicitly. This is because in 1.1 users can omit the storage class name if they want to use the default storage class in the Kubernetes (`pd-ssd` in GKE).

Reply:
I remembered that the `volumeBindingMode` of the default storage class `pd-ssd` of GKE is immediate, if that's the case, we may not use it, please help confirm.

Reply:
reverted